source/fuzz/fuzzer_pass_construct_composites.cpp

```diff
@@ -172,7 +158,8 @@ void FuzzerPassConstructComposites::RecordAvailableInstruction(
   if (type_id_to_available_instructions->count(inst->type_id()) == 0) {
```

Comment:
This part can also be removed. I think you can remove this method entirely and just use the newly added line.

Reply:
Done - thanks!